torch/utils/_pytree.py

```diff
@@ -190,9 +190,9 @@ def tree_unflatten(values: List[Any], spec: TreeSpec) -> PyTree:
 
     return unflatten_fn(child_pytrees, spec.context)
 
-def tree_map(fn: Any, pytree: PyTree) -> PyTree:
+def tree_map(fn: Any, pytree: PyTree, *rest: Any) -> PyTree:
     flat_args, spec = tree_flatten(pytree)
-    return tree_unflatten([fn(i) for i in flat_args], spec)
+    return tree_unflatten([fn(*i) for i in zip(flat_args, *rest)], spec)
```

Comment:
Would renaming `i` to `flat_arg` make more sense?

Reply:
Sure, updated it.